log.go

```diff
@@ -473,8 +473,8 @@ func (l *slog) SetLevel(level Level) {
 }
 
 // Disabled is a Logger that will never output anything.
-var Disabled Logger
+var Disabled Logger = &slog{lvl: LevelOff, b: NewBackend(ioutil.Discard)}
 
-func init() {
-	Disabled = &slog{lvl: LevelOff, b: NewBackend(ioutil.Discard)}
```

Comment:
fyi, this was done just to remove some ugly from the godoc page which would otherwise show the variable initialized to an unexported type, which isn't super helpful